code/modules/power/supermatter/supermatter.dm

```diff
@@ -613,7 +613,10 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 			if(!istype(delamination_strategy, /datum/sm_delam/cascade))
 				return
 
-			for(var/mob/living/lucky_engi as anything in mobs_in_area_type(list(/area/station/engineering/supermatter)))
+			for(var/mob/living/lucky_engi as anything in mobs_in_area_type(list(\
+				/area/station/engineering/supermatter,\
+				/area/station/engineering/supermatter/room,\
+				/area/station/engineering/supermatter/waste)))
```

Comment:
Did someone come to you with this issue? Because `mobs_in_area_type()` does an `istype()` check instead of an exact type comparison, that means it should work with subtypes and it does
![image](https://github.com/tgstation/tgstation/assets/47710522/d51de2fd-c56f-4ed0-a89c-d0773c2e644c)
![image](https://github.com/tgstation/tgstation/assets/47710522/f38fecd8-a833-4249-a387-12b2b9eb6d53)

Reply:
Yeah a person recorded themselves saving the SM and not getting the achievement, in a room with a bunch of others and no one else got it either.